examples/collector-gateway-only-advanced/README.md

```diff
@@ -0,0 +1,9 @@
+# Example of chart configuration
+
+## Enable OTel Collector in the gateway mode with more advanced options
+This configuration installs collector as a gateway deployment along with
+regular components. All the telemetry will be routed through this collector.
+By default, the gateway-mode collector deployed with 3 replicas with 4 CPU
+cores and 8Gb of memory each, but this can be easily changed as in this example.
```

Comment:
Maybe add commas when listing specs, so it would be "3 replicas, 4 CPU cores, and..."

Also, it looks like this doesn't match up with the YAML file below, which looks like it has 1 replica, 2 CPU cores. and a 4GB memory. Should this be updated to match the YAML or am I missing something?